src-electron/rest/file-ops.js

```diff
@@ -41,24 +41,31 @@ const studio = require('../ide-integration/studio-rest-api')
 function httpPostFileOpen(db) {
   return async (req, res) => {
     let { zapFilePath, ideProjectPath } = req.body
-    let name = ''
+    if (req.body.search?.includes('filePath=%2F')) {
+      ideProjectPath = dbEnum.sessionKey.ideProjectPath
+      zapFilePath = req.body.search?.split('filePath=')
```

Comment:
I see the "search" variablle is being reparsing the string again here.
Why isn't the content of the "filePath" passed as "zapFilePath" instead?

Reply:
had to redo how the rest API which opens files is called